frontend/src/components/Dashboard/Dashboard.jsx

```
@@ -0,0 +1,222 @@
+  return (
+    <div
+      className={`${
+        isCollapsed ? 'w-20' : 'w-64'
+      } bg-white shadow-lg fixed h-full transition-all duration-300`}
+    >
+      <div className="p-6 flex items-center justify-between">
+        <h1 className="text-2xl font-bold">{isCollapsed ? 'AH' : 'Amearhealth'}</h1>
+        <button
+          onClick={toggleSidebar}
+          className="text-gray-600 hover:text-gray-800 focus:outline-none"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)